### ISSUE TITLE
TypeError: 'NoneType' object is not iterable

### ISSUE THREAD

**Issue by mindvox** [k4m4/kickthemout#83]
Cloned the repo, installed the dependencies with `pip` and launched. I then get the following error.

```bash
ERROR: Default Interface MAC Address could not be obtained. Please enter MAC manually.
kickthemout> Enter MAC Address (MM:MM:MM:SS:SS:SS): **:**:**:**:**:**
```

So I enter the MAC of `eth0` and then the following is returned.

```bash
Traceback (most recent call last):
  File "/home/karlbateman/kickthemout/kickthemout.py", line 411, in <module>
    scanNetwork()
  File "/home/karlbateman/kickthemout/kickthemout.py", line 121, in scanNetwork
    regenOnlineIPs()
  File "/home/karlbateman/kickthemout/kickthemout.py", line 99, in regenOnlineIPs
    for host in hostsList:
TypeError: 'NoneType' object is not iterable
```

I have a Docker and VPN network adapter, hence the reason to enter the MAC manually I guess.

**Python** 2.7.9
**PIP** 1.5.6
**OS** Debian 8 (Jessie) amd64

**Comment by TheMelvin** [comment]
Same problem here...

**Comment by k4m4** [comment]
We'll have to look in to this and let you know when we've fixed it.
Thanks!

**Comment by k4m4** [comment]
Hi there @mindvox. Are you still experiencing this issue with v2.0? Thanks!

**Comment by td0** [comment]
I still experiencing this error. I am using mac OS high sierra. Installed brew libdnet and nmap since long ago. I ran sudo pip3 ... command, and getting this:
![image](https://user-images.githubusercontent.com/4467386/36519062-dbe62984-17bb-11e8-88f9-1bdddcb7de89.png)

opened python3 to try to find the error. Copy paste line by line. I get an error on line 32 which is
`from scapy.all import *`. it gave me this on python3 console:
![image](https://user-images.githubusercontent.com/4467386/36519123-3fe161ba-17bc-11e8-9359-3f59cdd16b4c.png)

Also, before running it on mac's system python, I ran it using pyenv and modify the code a bit (so I shouldn't need sudo command). It has the same error as using system python3 with sudo
